setup.py

```diff
@@ -433,6 +382,74 @@ def __init__(self, dist):
     def add(self, ext):
         self.extensions.append(ext)
 
+    def addext(self, ext, *, update_flags=True):
+        """Add extension with Makefile MODULE_{name} support
+        """
+        if update_flags:
+            self.update_extension_flags(ext)
+
+        state = sysconfig.get_config_var(f"MODULE_{ext.name.upper()}")
+        if state == "yes":
+            self.extensions.append(ext)
+        elif state == "disabled":
+            self.disabled_configure.append(ext.name)
+        elif state == "missing":
+            self.missing.append(ext.name)
+        elif state == "n/a":
+            # not available on current platform
+            pass
+        else:
+            # not migrated to MODULE_{name} yet.
```

Comment:
Perhaps print an info message?

Reply:
Good thinking, I plan to add an error message after we have ported all stdlib modules to new system.